content/200-concepts/100-components/01-prisma-schema/index.mdx

```diff
@@ -136,6 +136,31 @@ Introspecting based on datasource defined in prisma/schema.prisma …
 Run prisma generate to generate Prisma Client.
 ```
 
+## Accessing environment variables from the schema
+
+Environment variables can be accessed using the `env` function:
+
+```prisma
+datasource db {
+  provider = "postgresql"
+  url      = env("DATABASE_URL")
+}
+```
+
+In version 2.7.0 and later, you can expand variables using the format specified by [dotenv-expand](https://github.com/motdotla/dotenv-expand). This is useful when dealing with PaaS or similar products. such as Heroku):
+
+```
+# .env
+DATABASE_URL=postgresql://test:test@localhost:5432/test
+DATABASE_URL_WITH_SCHEMA=${DATABASE_URL}?schema=public
+```
+
+There are a few limitations with `env` at the moment:
```

Comment:
This is not going away, so "at the moment" could give a wrong indication. 
Maybe we should better list _where_ it can actually be used (datasource.url, generator.binaryTargets - not sure about others).